app/javascript/geoblacklight/openlayers/inspection.js

```diff
@@ -0,0 +1,45 @@
+import { linkify } from "geoblacklight/leaflet/utils";
+
+export const pmTilesInspection = (map) => {
+  // add crosshair class for map items
+  map.on('pointermove', function(e){
+    var pixel = map.getEventPixel(e.originalEvent);
+    var hit = map.hasFeatureAtPixel(pixel);
+    map.getViewport().style.cursor = hit ? 'crosshair' : '';
+  });
+
+  const mapTarget = map.getTargetElement();
```

Comment:
`mapTarget` doesn't seem to get used for anything, from what I can tell?

Reply:
Good catch. That was some leftover code. Removed.